src/main/java/com/kryptnostic/api/v1/search/DefaultSearchClient.java

```diff
@@ -97,15 +96,4 @@ private SearchRequest generateSearchRequest( List<String> tokens ) {
         }
         return Lists.newArrayList( tokens );
     }
-
-    @Override
-    public SearchResultResponse nextPage( SearchResultResponse response ) {
```

Comment:
I'm not really sure what's going on in this PR, but removing pagination seems like a bad thing... are you dropping pagination support?